test/interval_tests/rounding.jl

```diff
@@ -34,6 +34,12 @@ setrounding(Interval, :tight)
     @test sin(x) == Interval(0.47942553860420295, 0.479425538604203)
 end
 
+setrounding(Interval, :emulation)
+@testset "Tight rounding using RoundingEmulation.jl" begin
+    @test rounding(Interval) == :emulation
+    @test sin(x) == Interval(0.47942553860420295, 0.479425538604203)
```

Comment:
I don't think `sin` is a good test here, since that doesn't actually use the `emulation` functions at all?

Reply:
Maybe you can add some of the tests from #215 instead.

Reply:
Yes, I added tests for `tiny` and `huge` of #215.